Transrate `contributing/how-to-file-an-issue` (#197)
gatsbyjs/gatsby-ja: docs/contributing/how-to-file-an-issue.md

---
title: Issue を提出する方法
---

バグ報告や機能の提案、Pull Request の送信は、GitHub の [Issues](https://github.com/gatsbyjs/gatsby/issues) より行うことができます。

Issue を迅速に解決したい場合は、以下を Issue に含めてください。

- Gatsby, Node.js, OS の各バージョン

> - `gatsby-config.js` , `package.json` の内容、およびあなたの行った変更に応じて `gatsby-node.js` , `gatsby-browser.js`, `gatsby-ssr.js` の内容

Reviewer: 読点は基本的に全角を使うようにしています。:pray:
```suggestion
- `gatsby-config.js`、`package.json` の内容、およびあなたの行った変更に応じて `gatsby-node.js` 、`gatsby-browser.js`、`gatsby-ssr.js` の内容
```

> 個人の課題のサポートのためには Issue を利用しないでください。その場合は、[Stack Overflow](https://stackoverflow.com/questions/ask?tags=gatsby) の**gatsby** タグや [Gatsby Discord](https://gatsby.dev/discord) を利用するのが良いでしょう。

Reviewer: ```suggestion
個人の課題のサポートのために Issue を利用するのは控えてください。[Stack Overflow](https://stackoverflow.com/questions/ask?tags=gatsby) の **gatsby** タグや [Gatsby Discord](https://gatsby.dev/discord) で質問すると良いでしょう。
```

> ### Issue に関する特記事項

Reviewer: このセクションは、既存の Issue に対する特記事項ぽいので、「既存の」と言う記載があるとわかりやすいかも。
```suggestion
### 既存の Issue に関する特記事項
```

Reviewer: と思ったけど3番目でIssue が無い場合について書いてあるので、ここは「既存」付けなくても良いか‥？

> Issue があなたに影響する場合は、以下リストの一番上から初めて、できるだけ多くのタスクを完了させてください。

Reviewer: ```suggestion
既存の Issue をあなたも踏んでいる場合は、以下の手順を上から初めて、できるだけ多くの手順をやってみましょう。
```

Author: @Naturalclar 

[質問] 
`Issueを踏んでいる` という表現を、あまり聞いた事のですがこちら一般的なのでしょうか？？？（  `バグを踏むと同じなのかな？）

`既存の Issue があなたにも（関係 / 影響 ）している場合は、` でも良いのかなと思ったりしました

Reviewer: 確かに、そうですね！「バグを踏む」みたいな感覚でIssueを踏むと書きましたが、「既存の Issue があなたにも影響している場合〜」のほうがわかりやすいと思います！

Author: ご確認ありがとうございます!

Sugestいただいたものをコミットした上で、
https://github.com/gatsbyjs/gatsby-ja/pull/197/commits/c22b79db2a9cccba1c9384f23c4c8ddd82beb6d0 で該当の部分だけ `既存の Issue があなたにも影響している場合 ` に書き換えました

> 1. Issue がある場合は、リアクションや課題の詳細を追加することであなたに影響していることを示します。

Reviewer: ```suggestion
1. 既存の Issue に対して、リアクションや課題の詳細を追加することであなたにも影響していることを示します。
```

> 2. Issue があり、あなたが詳細を追加できる場合は、バグがどのように影響しているかをコメントしたり、可能であればバクの回避策を書きます。

Reviewer: ```suggestion
2. 既存の Issue に対して、あなたが詳細を追加できる場合は、バグがどのように影響しているかをコメントしたり、可能であればバグの回避策を書きましょう。
```

> 3. Issue がない場合は、何が起こっているかについて詳細な説明を書いてください。可能であれば問題を再現できる Gatsby サイトへのリンクを追記するか、[再現可能なテストケース](/contributing/how-to-make-a-reproducible-test-case/)を作成します。

Reviewer: ```suggestion
3. 既存の Issue がない場合は、新しく Issue を起票し、何が起こっているかについて詳細な説明を書いてください。可能であれば問題を再現できる Gatsby サイトへのリンクを追記するか、[再現可能なテストケース](/contributing/how-to-make-a-reproducible-test-case/)を作成しましょう。
```

Author: なるほど!!既存のIssueがない場合のことだったのですね!!

> 3. Issue がない場合は、何が起こっているかについて詳細な説明を書いてください。可能であれば問題を再現できる Gatsby サイトへのリンクを追記するか、[再現可能なテストケース](/contributing/how-to-make-a-reproducible-test-case/)を作成してください。

Author: `If there _is not_ an Issue` を `Issue がない場合` と直訳してしまったのですが、意味が通らないので違う気がしています。。

Author: `Issueでない場合` なのかな？

Author: `既存のIssueがない場合は、、` だったのですね

> 4. バグを修正することを申し出ます。（何か詰まった場合はメンテナーに助けを求めると良いでしょう。オープンソースのメンテナーは貢献者をサポートしたいのです）

Author: `何か詰まった場合はメンテナーに助けを求めると良いでしょう。オープンソースのメンテナーは貢献者をサポートしたいのです）` 意訳しすぎている気もしています

Reviewer: 良いと思います！「詰まったら‥」ではなく「わからないところは‥」のほうが、気軽に聞けそうな印象与えられるかな？と思いました(imo)

> 4. バグを修正することを申し出ます。（何か詰まった場合はメンテナーに助けを求めると良いでしょう。オープンソースのメンテナーは貢献者をサポートしたいのです）

Reviewer: 意訳ですが、前提条件があったほうが読みやすいと感じました。
原文にPRしても良いかも。
```suggestion
4. もし、自分でバグで修正してみたいと思ったら、バグを修正することを申し出ましょう。（何か詰まった場合はメンテナーに助けを求めると良いでしょう。オープンソースのメンテナーは貢献者をサポートしたいのです）
```

> 4. もし、自分でバグを修正してみたいと思ったら、バグを修正することを申し出ましょう。（わからないところはメンテナーに助けを求めてみましょう。オープンソースのメンテナーは貢献者をサポートしたいのです。）

Reviewer: すみません、ここ自分の Suggest で Lint エラーが出てしまったぽいです
```suggestion
4. もし、自分でバグを修正してみたいと思ったら、バグを修正することを申し出ましょう（わからないところはメンテナーに助けを求めてみましょう。オープンソースのメンテナーは貢献者をサポートしたいのです）。
```

Author: なるほど〜。 `()` が関わる句読点の取り扱いはこうなるのですね。

> 5. よく検討されたテスト済みの[Pull Request を作成](/contributing/how-to-open-a-pull-request/)します。

Reviewer: 括弧があると lint に catch されませんが、半角英数の周りにはスペースを入れるようにしています。
```suggestion
5. よく検討されたテスト済みの [Pull Request を作成](/contributing/how-to-open-a-pull-request/)します。
```
